src/openrct2-ui/UiContext.cpp

```diff
@@ -963,6 +942,32 @@ class UiContext final : public IUiContext
             return;
         }
     }
+
+    InputEvent GetInputEventFromSDLEvent(SDL_Event& e)
+    {
+        InputEvent ie;
+        ie.DeviceKind = InputDeviceKind::Keyboard;
+        ie.Modifiers = e.key.keysym.mod;
+        ie.Button = e.key.keysym.sym;
+
+        // Handle dead keys
+        if (ie.Button == SDLK_SCANCODE_MASK)
```

Comment:
This could alternatively be written as `if (ie.Button == (SDLK_SCANCODE_MASK | 0))`, which is a little bit clearer.

Reply:
```suggestion
        if (ie.Button == (SDLK_SCANCODE_MASK | 0))
```

Reply:
@AaronVanGeffen It's because the `0` part is the most important bit.